bin/propolis-server/src/lib/vm/objects.rs

```diff
@@ -344,9 +344,11 @@ impl VmObjectsLocked {
 
     /// Pauses all of a VM's devices.
     async fn pause_devices(&self) {
-        self.for_each_device(|name, dev| {
-            info!(self.log, "sending pause request to {}", name);
-            dev.pause();
+        tokio::task::block_in_place(|| {
```

Comment:
maybe worth a comment noting why this is considered a blocking op? up to you, though.

Reply:
Will do.